static/js/util/sorting.js

```diff
@@ -1,9 +1,10 @@
 // @flow
 import moment from 'moment';
+import _ from 'lodash';
 
-import type { WorkHistoryEntry } from '../flow/profileTypes';
+import type { WorkHistoryEntry, EducationEntry } from '../flow/profileTypes';
 
-export function resumeOrder(entries: WorkHistoryEntry[], dateFieldName: string): WorkHistoryEntry[] {
+export function resumeOrder(entries: any, dateFieldName: string): any {
```

Comment:
I think you can declare an array of union types, but I wasn't able to get that to work because then flow can't make guarantees about the type of the return value, and it gives all these errors about not finding the properties on `WorkHistoryEntry` on an `EducationEntry`.